src/hypothesis/extra/datetime.py

```diff
@@ -123,3 +123,26 @@ def times(allow_naive=None, timezones=None):
 
 def datetime_to_time(dt):
     return dt.timetz()
+
+SECS_IN_DAY = 3600 * 24
+MICROS_IN_SEC = 1000000
+MICROS_IN_DAY = MICROS_IN_SEC * SECS_IN_DAY
+class TimedeltaStrategy(SearchStrategy):
+
+    def __init__(self, min_value=dt.timedelta.min, max_value=dt.timedelta.max):
```

Comment:
That's correct. I added some assertions in my commit to make that clearer and to remove any room for error if the user passes the wrong types.